apps/meteor/app/utils/lib/templateVarHandler.js

```diff
@@ -1,11 +1,6 @@
-import { Meteor } from 'meteor/meteor';
+import { Logger } from '../../../server/lib/logger/Logger';
 
-let logger;
-
-if (Meteor.isServer) {
-	const { Logger } = require('../../../server/lib/logger/Logger');
-	logger = new Logger('TemplateVarHandler');
-}
```

Comment:
Not a problem worth reverting for but this file should've been moved to a "server" folder now that it only works in the server.